11.05.2020_homework_java_collections/src/list/task2/Name.java

```diff
@@ -0,0 +1,19 @@
+package list.task2;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Name {
+
+    public static List<String> removeNameWithLength4(List<String> namesList) {
+        List<String> newNameList = new LinkedList<>(namesList);
+        for (int i = 0; i < newNameList.size(); i++) {
+            if (newNameList.get(i).length() == 4) {
+                newNameList.remove(i);
```

Comment:
This is not a right/safe way of removing elements from the list. Let's check why in the class.